### PR TITLE
add RheuX.md

### PR DESCRIPTION
**Pull request by RheuX**
**Important:** *Do not tick a checkbox if you haven’t performed its action.*

### Before Creating the Pull Request

- [x] Verify the changes are on a new branch other than `master` – it should say "compare: branch-name".
- [x] Start the pull request title with lowercase letters, e.g., `add githubusername.md`.
- [x] Add issue number to the end of the pull request title when applicable, e.g., `update mi-faq.md (fixes #3264)`.

### After Creating the Pull Request

- [x] Go to "Commits" tab, make sure the commit username is clickable and linked to your GitHub account correctly.
- [x] Review the "Files changed" tab to ensure there are no unnecessary files or changes included in the pull request.
- [x] Verify that the raw.githack preview link is included in the description.
  - [x] Preview the MDwiki rendered changes using the raw.githack link. Ensure it displays as expected without any errors.
- [x] Drop a link to this pull request in our discord channel.

### Description, Screenshots and/or Screencast

fixes #

### Raw.Githack Preview Link

[RheuX Raw.Githack Preview Link](https://raw.githack.com/RheuX/RheuX.github.io/add-Rheux-profile/index.html#!pages/vi/profiles/RheuX.md)

### CODE REVIEW

File: pages/vi/profiles/RheuX.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Hello 👋 
+
+## Basic Info
+### My name is Axel Nicholas Lorens
```
Comment (xyb994) [inline pages/vi/profiles/RheuX.md:4]: Also please see https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/3383#discussion_r1793841981

**Review by xyb994 — COMMENTED** [review]
Looking good! 🎉
Could you also include the OS you use in the profile?

File: pages/vi/profiles/RheuX.md (hunk 1 above)
Comment (RheuX) [reply pages/vi/profiles/RheuX.md]: Hi, I did some changes with the semantics, I only use it when necessary, if I need to bring a new topic.

Also, I'm not sure what you mean by `include the OS`, Can you see the OS that I put in my previous and current .md? I put down a Macbook and Windows 11 for my OS. Unless you're referring to another OS term that I'm not sure about

**Review by xyb994 — APPROVED** [review]

File: pages/vi/profiles/RheuX.md (hunk 1 above)
Comment (xyb994) [reply pages/vi/profiles/RheuX.md]: > Hi, I did some changes with the semantics, I only use it when necessary, if I need to bring a new topic.
> 
> Also, I'm not sure what you mean by `include the OS`, Can you see the OS that I put in my previous and current .md? I put down a Macbook and Windows 11 for my OS. Unless you're referring to another OS term that I'm not sure about

I apologize; I missed that you already mentioned your OS in the document. It typically appears as a list early in the document, which is probably why I overlooked it.

**Review by dogi — APPROVED** [review]

**Pull request merged**